prow/cmd/hook/Dockerfile

```diff
@@ -15,7 +15,7 @@
 FROM alpine:3.4
 MAINTAINER spxtr@google.com
 
-RUN apk add --no-cache ca-certificates && update-ca-certificates
+RUN apk add --no-cache git ca-certificates && update-ca-certificates
```

Comment:
Yes. There are a few problems with that that I can add to the doc. One problem is that `git-sync` watches one repo specified as a flag, whereas we don't know what repos to watch until we need them. It's trickier to add multiple repos.